tests/client/amo/components/TestAddonDetail.js

```diff
@@ -291,19 +296,31 @@ describe('AddonDetail', () => {
         ...fakeAddon,
         type: ADDON_TYPE_THEME,
       },
-      clientSupportsAddons: () => true,
+      isCompatibleWithUserAgent: () => ({ compatible: true }),
     });
     const button = rootNode.querySelector('.AddonDetail-theme-header-label');
     assert.equal(button.disabled, false);
   });
 
+  it('disables install switch for unsupported clients', () => {
+    const rootNode = renderAsDOMNode({
+      addon: {
```

Comment:
`addon` does not seem relevant to this test so you can delete it and rely on the default